docs/github-action.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: GitHub Action
+permalink: /github-action/
+nav_order: 6
+---
+
+# GitHub Action
+
+{: .no_toc }
+
+OSV-Scanner is offered as a GitHub Action. We currently have two different GitHub Actions:
+
+1. An action that performs a vulnerability scan on a [regular schedule](./github-action.md#scheduled-scans). 
+2. An action that triggers a scan with each [pull request](./github-action.md#scans-on-prs) and will only check for new vulnerabilities introduced through the pull request. 
+
+## Scheduled scans
+
+Regularly scanning your project for vulnerabilities can alert you to new vulnerabilities in your dependency tree. This GitHub Action will scan your project on a set schedule and report all known vulnerabilities. 
+
+### Instructions
+
+In your project repository, create a new file `.github/workflows/osv-scanner-scheduled.yml`. 
+
+Include the following in the [`osv-scanner-scheduled.yml`](https://github.com/google/osv-scanner/blob/main/.github/workflows/osv-scanner-scheduled.yml) file:
+
+```yml
+name: OSV-Scanner Scheduled Scan
+
+on:
+  schedule:
+    - cron: '12 12 * * 1'
+# Change "main" to your default branch if you use a different name, i.e. "master"
+  push:
+    branches: [ "main" ]
```

Comment:
@another-rex Why are double quotes used here,  but lines 70 and 72 don't use them?

Reply:
There's not really a reason, both are valid I believe. let's make it consistent and remove the quotes here.
```suggestion
    branches: [ main ]
```